setup.py

```diff
@@ -52,7 +52,6 @@ def get_data_files():
                                                         'voila-gridstack/static/voila-gridstack.js',
                                                         'voila-gridstack/static/voila-gridstack.css',
                                                         'voila-gridstack/static/gridstack.js',
-                                                        'voila-gridstack/static/gridstack.jqueryUI_require.js'
```

Comment:
:tada:

Reply:
@jtpio should we keep the listing of gridstack.js above?

Reply:
Looks like it's still needed for the classic notebook extension:

https://github.com/voila-dashboards/voila-gridstack/blob/29e9569263cbfc8ac7aef7071cd73cbb09dcc2d3/voila-gridstack/static/extension.js#L13

Reply:
But you delete the gridstack.js file from the repository?

Reply:
It's not deleted by updated with the new (minified) version.

Reply:
Sorry, I am confused. I see the file deleted in the PR.